containers/podman-desktop
# chore: update extension catalog screenshot

### What does this PR do?

The extension catalog was a bit out of date, updated to newer image.

Our UI has changed a bit - maybe this looks out of place compared to the others on the page? I could do the others as well.

### Screenshot / video of UI

See commit.

### What issues does this PR fix or reference?

Fixes #9342.

### How to test this PR?

`pnpm website:dev` and go to Documentation > Extensions > Installing to see the image in context.
